src/package.xml

```diff
@@ -729,6 +744,7 @@
         <members>Account-HEDA Organization Layout</members>
         <members>Address__c-HEDA Address Layout</members>
         <members>Affiliation__c-HEDA Affiliation Layout</members>
+        <members>Attendance_Event__c-Attendance Event Layout</members>
```

Comment:
@xinnanlisf @lifewithryan This is interesting -- I would say that we should be consistent here with the naming and call it the "HEDA Attendance Event Layout" ... thoughts? We might want to discuss in Quip/standup.

Reply:
Yep, good catch!